Symfony/CS/Fixer/PSR2/ClassDefinitionFixer.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\PSR2;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * Fixer for part of the rules defined in PSR2 ¶4.1 Extends and Implements.
+ *
+ * @author SpacePossum
+ */
+final class ClassDefinitionFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+        for ($index = $tokens->getSize() - 1; $index > 0; --$index) {
+            if (!$tokens[$index]->isClassy()) {
+                continue;
+            }
+
+            $this->fixClassDefinition($tokens, $index, $tokens->getNextTokenOfKind($index, array('{')));
+        }
+
+        return $tokens->generateCode();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDescription()
+    {
+        return 'White space around the key words of a class, trait or interfaces definition should be one space.';
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $start  Class definition token start index
+     * @param int    $end    Class definition token end index
```

Comment:
From both phpdoc and your explanation i would say it is the index of last `}` in the following code:
```php
class Foo extends Bar implements Baz {
    public function xxx()
    {
    }
}
```